microsoft/atlas-design
# add button, buttons, and related mixins

Task: task-432750

Link: preview-208

#132 Discussion

Update buttons to fit new designs. Heavily tied to icons, so updated that as well.

https://design.docs.microsoft.com/pulls/208/components/button.html
https://design.docs.microsoft.com/pulls/208/components/icon.html

## Testing

1. Open pages.
2. Use a color contrast tool like accessiblity insights or tota11y to check the contrast.
3. Check states, ensure they match designs.
4. Read code!
